source/guides/dashboard/slack-integration.md

```diff
@@ -0,0 +1,112 @@
+
+# Per-organization configuration
+
+## Add additional Slack channels
+
+You can have the Cypress Dashboard post run results to an additional channels. To add a channel:
+
+1. Navigate to the **Integrations** page for the organization with the installed Slack integration.
+1. Within the Slack integration, click **Configure**.
+1. Click **Add Slack Channel**.
+1. You'll see a popup window that allows you to choose the channel to associate with the organization. The Cypress Dashboard will post run results for all projects in your organization to the new Slack channel.
+
+## Set notification preferences
+
+By default, the Cypress Dashboard will post a Slack message to each configured channel only for failing runs. If you'd like to change these preferences:
+
+1. Navigate to the **Integrations** page for the organization with the installed Slack integration.
+1. Within the Slack integration, click **Configure**.
+1. Under **Notifications**, select your preference for each Slack channel:
+  - **All runs**: will notify on all runs (including passed)
+  - **Failed runs only**: will only notify on runs with a fail status.
+
+## Mute a channel
+
+If you want the Cypress Dashboard to temporarily stop posting Slack messages to a certain channel, you can **Mute** that channel. This allows you to easily pause and resume notifications for a specific channel without losing the configuration you’ve put in place.
+
+1. Navigate to the **Integrations** page for the organization with the installed Slack integration.
+1. Within the Slack integration, click **Configure**.
+1. Under **Actions**, select your **Mute** for each Slack channel you want muted.
+
+## Remove a Slack channel
+
+You can have the Cypress Dashboard stop posting notifications to a channel. You can remove all Slack channels if you’d prefer to disable global notifications altogether in favor of per-project notifications.
+
+1. Navigate to the **Integrations** page for the organization with the installed Slack integration.
+1. Within the Slack integration, click **Configure**.
+1. Under **Actions**, select your **Delete** for each Slack channel you want deleted.
+
+# Per-project configuration
+
+If your organization has multiple teams working on separate projects, you can tailor the Slack notifications of each project to match your teams' needs.
+
+## Add a new Slack channel
+
+You can have the Cypress Dashboard post run results for a specific project to an additional channel.
+
+1. Select your organization in the organization switcher.
+  {% imgTag /img/dashboard/select-cypress-organization.png "Select an organization" width-600 %}
+1. Select the project you wish to integrate with a GitHub repository.
```

Comment:
This is referring to GitHub. Where did that come from?

Reply:
This was a mistake from copying over from GitHub doc, since we have the same steps for entering the Integrations pages.